cases/src/org/commcare/cases/instance/CaseInstanceTreeElement.java

```diff
@@ -57,12 +51,9 @@
      */
     final boolean reportMode;
 
-    public CaseInstanceTreeElement(AbstractTreeElement instanceRoot, IStorageUtilityIndexed storage, String[] caseIDs) {
-        this(instanceRoot, storage, false);
-        this.caseRecords = caseIDs;
-    }
```

Comment:
Appears to be unused in commcare, formplayer, and touchforms. Removed code associated with `caseRecords` because this is the only place it is set